### DISCUSSION TITLE
Need a galaxy brain!

### DISCUSSION THREAD

**Discussion by Guardiaans** [Schweinepriester/github-profile-achievements#462] [Q&A] [answered]
Looking to get a galaxy brain!

**Comment by fcasamento** [comment]
Hi @Guardiaans. 

How are you? I suggest two places for you to answer people and won the badge.
Here are my suggestions:

https://github.com/STRINGLABITSOLUTIONS/Bhojon/discussions
https://github.com/orgs/community/discussions

I can create a short question and you answer, to have your first badge you have to answer two questions and the person marks that answer as "correct".

**Comment by fcasamento** [comment, marked as answer]
@Guardiaans  If my answer helped you, you could mark it as answered. Please.

**Comment by haithamaouati** [comment]
Done! Could I get Galaxy Brain?
